lock/src/main/java/org/infinispan/lock/api/impl/ClusteredLockMetadataFileFinder.java

```diff
@@ -0,0 +1,19 @@
+package org.infinispan.lock.api.impl;
+
+import org.infinispan.factories.components.ModuleMetadataFileFinder;
+import org.kohsuke.MetaInfServices;
+
+/**
+ * Metatada file name, needed for dependency injection.
```

Comment:
typo: `Metatada` => `Metadata`

Reply:
Metatada is much more fun Pedro ! 👍 thanks for all the typo reviews !